generator.rb

```diff
@@ -0,0 +1,72 @@
+def is_vowel?(letter)
+    result = false
+    if (letter == "a"|| letter == "e" || letter == "i" || letter == "o" || letter == "u")
+        result = true
+    end
+    return result
+end
+
+    #test1 = "i"
+    #puts "Is #{test1} a vowel?"
+    #puts is_vowel?(test1)
+
+    #test2= "v"
+    #puts "Is #{test2} a vowel?"
+    #puts is_vowel?(test2)
+
+def run_generator
+    puts "Please enter your first word:"
+    first_word = gets.chomp.downcase
+    until first_word.length >= 2
+        puts "Please enter a word 2 character or longer"
+        first_word = gets.chomp.downcase
+    end
+
+    first_word_array = []
+    first_word.each_char{|x| first_word_array.push(is_vowel?(x))}
+    until first_word_array.include?(true) == true
+        puts "Please enter a word with a vowel:"
+        first_word = gets.chomp.downcase
+        first_word_array = []
+        first_word.each_char{|x| first_word_array.push(is_vowel?(x))}
+    end
+
+    puts "Please enter your second word:"
+    second_word = gets.chomp
+    until first_word.length >= 2
+        puts "Please enter a word 2 character or longer"
+        first_word = gets.chomp
```

Comment:
Copy paste error here. It looks like you meant to do `until second_word.length >=2`